kivy/tests/test_environ_cli.py

```diff
@@ -0,0 +1,54 @@
+# coding=utf-8
+
+import itertools
+from functools import partial
+from os import environ
+import shlex
+import subprocess
+import sys
+
+import pytest
+
+ENV_NAME="KIVY_NO_ARGS"
+EXPECTED_STR = "Kivy Usage"
+
+
+TRUTHY = {'true', '1', 'yes'}
+FALSY = {'false', '0', 'no', "anything-else"}
+
+SAMPLE_VALUES = {
+    *TRUTHY, *FALSY
+}
+
+def _patch_env(**kw):
+    env = {
+        k: v
+        for k, v in environ.items()
+        if "kivy" not in k.lower()
+    }
```

Comment:
```suggestion
def _patch_env(*filtered_keys, **kw):
    env = {
        k: v
        for k, v in environ.items()
        if k not in filtered_keys
    }
```

And just add `KIVY_NO_ARGS` as args. Because I'd feel more comfortable if we removed specific env variables for specific tests (e.g. for new tests in the future), rather than all kivy variables for all tests.

Reply:
also added minimal kivy envs to implement test